Makefile

```diff
@@ -14,6 +14,24 @@ KUBECONFIG ?= $(HOME)/.kube/config
 GO_PACKAGES = ./cmd/... ./pkg/...
 GO_FILES := $(shell find $(shell $(GO) list -f '{{.Dir}}' $(GO_PACKAGES)) -name \*.go)
 
+COMMIT = $(shell git rev-parse HEAD)
```

Comment:
These commands force the user to build from a git clone, rather than (eg) a source-only tarball or git export.  I'm kind of ok with that (in particular tracking dirty state is useful), but an alternative would be to just make the build environment (eg: travis) set VERSION.

Happy to follow your preference here.

Reply:
This is slightly more automatic but we could allow override by env var. I was looking at https://github.com/go-semantic-release/semantic-release/, maybe it’s better to use it although a lot more moving pieces. WDYT?

Reply:
I don't really care - provided the git version tag ends up in the right places _somehow_ :P
Let's stick with what you've got here (using git commands).